ui/src/pages/backoffice/Series/SeriesDetails/SeriesDetails.js

```diff
@@ -54,7 +47,7 @@ export default class SeriesDetails extends Component {
         title: 'Literature in this series',
         content: (
           <Accordion.Content>
-            <div ref={this.anchors.documents} id="series-documents">
+            <div id="series-documents">
               <SeriesDocuments />
             </div>
```

Comment:
Cant we skip the extra div and  make like
`<SeriesDocuments id="series-documents"/>`
or 
```
<Accordeon.Content id="series-documents">
  <SeriesDocuments />
</Accordeon.Content>
```

Reply:
Accordeon :) this div was used there because not all props work in the `<Accordion`. However `id=` should appear in the final DOM. if possible I will fix it in next PR